Amazon.QLDB.Driver/driver/QldbDriver.cs

```diff
@@ -35,6 +35,7 @@ namespace Amazon.QLDB.Driver
     public class QldbDriver : IQldbDriver
     {
         private readonly QldbDriverBase<QldbSession> driverBase;
+        private readonly ISerializer serializer;
```

Comment:
Looking at this in both drivers, can't this live as a part of the QldbDriverBase?

Reply:
Good catch, moved `serializer` field to `QldbDriverBase`